src/components/DisplayNames/index.tsx

```diff
@@ -31,15 +49,21 @@ function DisplayNames({fullTitle, tooltipEnabled, textStyles, numberOfLines, sho
         );
     }
 
-    return (
-        <DisplayNamesWithToolTip
-            fullTitle={title}
-            displayNamesWithTooltips={displayNamesWithTooltips}
-            textStyles={textStyles}
-            numberOfLines={numberOfLines}
-            renderAdditionalText={renderAdditionalText}
-        />
-    );
+    if (shouldShowCaret) {
```

Comment:
Stylistically it doesn't look good, if you went with `return result;` approach why return in this if statement instead of using `result` variable (like in other ifs)?

Reply:
Hmm, I will look for a better approach. I may add a wrapper, which adds the `Icon`, and then wrap all the returns with it. But this also has some downsides.